front/app/api/authentication/reset_password/sendPasswordResetEmail.ts

```diff
@@ -0,0 +1,27 @@
+import fetcher from 'utils/cl-react-query/fetcher';
+import { TriggerResetPasswordProperties } from './types';
+
+const fetchResetPasswordEmail = (
+  requestBody: TriggerResetPasswordProperties
+) => {
+  return fetcher({
+    path: `/users/reset_password_email`,
+    action: 'post',
+    body: requestBody,
+  });
+};
+
+export default async function sendPasswordResetMail(email: string) {
+  // eslint-disable-next-line no-useless-catch
+  try {
```

Comment:
I _think_ you could leave the whole `catch` part out. So instead of catching the thrown error and throwing it again, we just let it throw (if that makes sense lol)

Reply:
Removing just the "catch" doesn't work :thinking: If we remove both the try and the catch, will it by default throw the error though still? :thinking:

Reply:
@luucvanderzee I've left the try/catch with the lint comment in for now, as it's unclear to me what we need here :thinking: Should we merge this for now into your branch, so we don't get any more merge conflicts?? It might be the case that you'll need to update some import paths after merging this in as well. :)